docs/routing.md

```diff
@@ -0,0 +1,128 @@
+
+## Basics
+
+1. Each Hotspot connects to a global Geo DNS Load Balancer [^1] (LB). The LB then redirects each Hotspot to a cluster of Helium Packet Routers (HPR) in its own region. Ex: If a Hotspot is in San Francisco it will get round-robin[^2] to the North America Cluster, if another is in Paris France, it will get round-robin to the EU cluster. *Note that each region may have a different number of servers to accomodate for load.*
+2. Each HPR gets its routing information from [Config Service](https://github.com/helium/oracles/tree/main/iot_config) (see [Routing Configuration](#routing-configuration)).
+3. Data is then routed to appropriate end-user's LoRaWAN Network Server (LNS)[^3].
+
+```mermaid
+flowchart LR
+    subgraph USA
+        H1(Hotspot 1)
+        HPRUS(HPRs US)
+    end
+    subgraph EU
+        H2(Hotspot 2)
+        H3(Hotspot 3)
+        HPREU(HPRs EU)
+        HPREU(HPRs EU)
+        
+    end
+    subgraph Global
+        LB(Load Balancer)
+        CS(Config Service)
+    end
+    subgraph LNSs
+        LNS1(LoRaWAN Network Server 1)
+        LNS2(LoRaWAN Network Server 2)
+    end
+    H1 --> LB
+    H2 --> LB
+    H3 --> LB
+    LB -->|Hotspot 1| HPRUS
+    LB -->|Hotspot 2| HPREU
+    LB -->|Hotspot 3| HPREU
+    HPRUS --> LNSs
+    HPREU --> LNSs
+    HPRUS <--> CS
+    HPREU <--> CS
+```
+
+## Configuration
+
+All of the device routing information is contained in the [Config Service](https://github.com/helium/oracles/tree/main/iot_config) (CS). The CS can be interacted via [CLI](https://github.com/helium/helium-config-service-cli) or directly using the [Proto APIs](https://github.com/helium/proto/blob/master/src/service/iot_config.proto).
+
+### OUI
+
+At the top of it all is the [Org (or OUI)](https://github.com/helium/proto/blob/master/src/service/iot_config.proto#L46) which defines a few things, like the `owner`, `payer`, and `delegate` keys [(see here)](https://github.com/helium/proto/blob/master/src/service/iot_config.proto#L49-L53).
+
+Upon request, an OUI is [created](https://github.com/helium/proto/blob/master/src/service/iot_config.proto#L651-L652) and assigned by the Helium Foundation. You can, at this point, purchase [Device Addresses or DevAddrs](https://github.com/helium/proto/blob/master/src/service/iot_config.proto#L27). If you would like to roam on the Helium network and already have your own [LoRaWAN Net ID](https://www.thethingsnetwork.org/docs/lorawan/prefix-assignments/) [see here](https://github.com/helium/proto/blob/master/src/service/iot_config.proto#L653-L654).
+
+### Route
+
+Routes tells each HPR exactly where to send data packets. To [create](https://github.com/helium/proto/blob/master/src/service/iot_config.proto#L671) a [Route](https://github.com/helium/proto/blob/master/src/service/iot_config.proto#L120) we recommend using the [Helium Config Service CLI](https://github.com/helium/helium-config-service-cli).
```

Comment:
[create] here is mistakenly linked to get route

Reply:
Fixed